docs/setup/install.md

```diff
@@ -18,6 +18,38 @@ VERSION=v0.1.0
 kubectl apply -f https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/manifests.yaml
 ```
 
+## Install a custom-configured released version
+
+To install a custom-configured released version of Kueue in your cluster, execute the following steps: 
+
+1. Download the release's `manifests.yaml` file using the following command:
+
+```shell
+VERSION=v0.1.0
+wget https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/manifests.yaml
+```
+2. Using your favourite editor, open `manifests.yaml` and edit the `kueue-manager-config` 
+ConfigMap manifest, specifically edit the `controller_manager_config.yaml` data entry which represents
+the default Kueue Configuration 
+struct ([v1alpha1@v0.1.0](https://pkg.go.dev/sigs.k8s.io/kueue@v0.1.0/apis/config/v1alpha1#Configuration)).
+The contents of the ConfigMap are similar to the following:
```

Comment:
These are the contents of a field of the ConfigMap, not the ConfigMap :)

I think we can just document the ConfigMap here, and leave the contents of the Configuration to a separate config doc.

Reply:
ok, but we still need to put a sample configuration, see how it looks now.